README.md

```diff
@@ -64,9 +45,32 @@ To have the content fit to the main area, use the `fit` class.
 </paper-header-panel>
 ```
 
-Use `mode` to control the header and scrolling behavior.
+### Mode
+
+Controls header and scrolling behavior. Options are `standard`, `seamed`, `waterfall`, `waterfall-tall`, `scroll` and
+`cover`. Default is `standard`.
+
+* `standard`: The header is a step above the panel. The header will consume the
+panel at the point of entry, preventing it from passing through to the
+opposite side.
+
+* `seamed`: The header is presented as seamed with the panel.
+
+* `waterfall`: Similar to standard mode, but header is initially presented as
+seamed with panel, but then separates to form the step.
 
-Styling header panel:
+* `waterfall-tall`: The header is initially taller (`tall` class is added to
+the header).  As the user scrolls, the header separates (forming an edge)
+while condensing (`tall` class is removed from the header).
+
+* `scroll`: The header keeps its seam with the panel, and is pushed off screen.
+
+* `cover`: The panel covers the whole `paper-header-panel` including the
+header. This allows user to style the panel in such a way that the panel is
+partially covering the header.
+
+
+### Styling header panel:
```

Comment:
It might be cool to get a nice table like this one for custom properties that are available:
https://github.com/PolymerElements/paper-input/blob/master/paper-input-container.html#L60-L68